index.html

```diff
@@ -396,17 +396,17 @@ <h3>
           "https://www.w3.org/TR/hr-time-2/#dom-domhighrestimestamp">DOMHighResTimeStamp</a>
           with a time value equal to the time immediately before the user agent
           sets the <a data-cite=
-          "!HTML/dom.html#current-document-readiness">current document
+          "HTML/dom.html#current-document-readiness">current document
           readiness</a> of the <a>current document</a> to <a data-cite=
-          "!HTML/parsing.html#the-end" title=
-          'interactive" document readiness'>"interactive"</a> [[!HTML]].
+          "HTML/parsing.html#the-end" title=
+          'interactive" document readiness'>"interactive"</a> [[HTML]].
         </p>
         <p data-dfn-for='PerformanceNavigationTiming'>
           The <dfn>domContentLoadedEventStart</dfn> attribute MUST return a
           <a href=
           "https://www.w3.org/TR/hr-time-2/#dom-domhighrestimestamp">DOMHighResTimeStamp</a>
           with a time value equal to the time immediately before the user agent
-          fires the <a data-cite="!HTML/parsing.html#the-end">DOMContentLoaded
+          fires the <a data-cite="HTML/parsing.html#the-end">DOMContentLoaded
```

Comment:
```suggestion
          fires the <a data-cite="HTML/parsing.html#event-domcontentloaded">DOMContentLoaded
```

The reader can then get to "#the-end" to understand where the event is fired.